frontend/webapp/reuseable-components/tab-list/index.tsx

```diff
@@ -66,22 +66,22 @@ const TABS = [
     selected: true,
     onClick: () => {},
   },
-  {
-    title: 'Service map',
-    icon: '/icons/overview/service-map.svg',
-    selected: false,
-    onClick: () => {},
-    disabled: true,
-    tooltip: 'Coming soon',
-  },
-  {
-    title: 'Trace view',
-    icon: '/icons/overview/trace-view.svg',
-    selected: false,
-    onClick: () => {},
-    disabled: true,
-    tooltip: 'Coming soon',
-  },
+  // {
+  //   title: 'Service map',
```

Comment:
I think it's best to delete it completely, it doesn't look like it's going to come back anytime soon.